ekotrace-capi/include/ekotrace.h

```diff
@@ -42,6 +42,8 @@ typedef struct ekotrace_instant {
     uint32_t event_count;
 } ekotrace_instant;
 
+typedef uint32_t chunked_report_token;
```

Comment:
Is this meant to be the same sized as the Rust structure? It probably is, but that seems really shaky.

Reply:
Yes, that's the point. We can/should add tests for this to be more sure.

Reply:
Yeah, I'd like to see a test for that, as it would be pretty catastrophic were that assumption to fail.

Reply:
Interestingly, the header size turns out to already be wrong, and the test that exercises it is still passing. Will add static assertions and think about what more to do for testing.